bson/src/main/org/bson/json/JsonWriterSettings.java

```diff
@@ -282,7 +291,7 @@ public String getIndentCharacters() {
     }
 
     /**
-     * The output mode to use.  The default value is {@code }JSONMode.RELAXED}.
+     * The output mode to use.  The default value is {@code }JSONMode.STRICT}.
```

Comment:
Is there any reason that all the changes in this class should not be reverted, with the exception of this one?

Reply:
I am not reverting correctly because the changes are showing. What is the proper way to revert so that the changes don't show?

Reply:
Reverting using IntelliJ was simple to do.